packages/flutter_web_plugins/lib/src/plugin_event_channel.dart

```diff
@@ -117,7 +117,7 @@ class _EventChannelHandler<T> {
 
   StreamSubscription<T>? subscription;
 
-  Future<ByteData>? handle(ByteData? message) {
+  Future<ByteData?>? handle(ByteData? message) async {
```

Comment:
The lint should not trigger in nullsafety code. I created https://github.com/dart-lang/linter/pull/2655

Reply:
your PR is merged and I have pulled the latest master.
and it's still showing the lint warning here! @a14n

Reply:
It's because no new linter version has been released and include in the dart-sdk. We need to wait this to happen.